### ISSUE TITLE
No data received since - 2017-07-04

### ISSUE THREAD

**Issue by LoveMyData** [planningalerts-scrapers/vincent#1]
Looks like the page is no longer exist

Ruby error:
```
/app/vendor/bundle/ruby/1.9.1/gems/mechanize-2.5.1/lib/mechanize/http/agent.rb:304:in `fetch': 404 => Net::HTTPNotFound for https://www.vincent.wa.gov.au/Your_Community/Whats_On/Community_Consultation/Planning_Applications -- unhandled response (Mechanize::ResponseCodeError)
 	from /app/vendor/bundle/ruby/1.9.1/gems/mechanize-2.5.1/lib/mechanize/http/agent.rb:949:in `response_redirect'
 	from /app/vendor/bundle/ruby/1.9.1/gems/mechanize-2.5.1/lib/mechanize/http/agent.rb:299:in `fetch'
 	from /app/vendor/bundle/ruby/1.9.1/gems/mechanize-2.5.1/lib/mechanize.rb:407:in `get'
 	from scraper.rb:7:in `<main>'
```

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_d959)

**Comment by LoveMyData** [comment]
New link but it no longer provide `council_reference`

https://www.vincent.wa.gov.au/consultations/

**Comment by MichaelBone** [comment]
Resolved by commit 499b77e.
